### PR TITLE
[UXE-5746] refactor: change is_active to active on Edge Firewall Rules Engine

### PR DESCRIPTION
**Pull request by pauloSF0**
### What is the new behavior introduced by this PR?
refactor: change is_active to active

### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.
Captura de Tela 2024-11-25 às 14 14 10

### Does it have a link on Figma?

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [ ] Application responsiveness was tested to different screen sizes
- [x] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

**Review by aloisio-m-bastian — DISMISSED** [review]

File: cypress/e2e/domains/edit-domain.cy.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ describe('Domains spec', { tags: ['@dev3'] }, () => {
     cy.get(selectors.domains.fieldTextInput).clear()
     cy.get(selectors.domains.fieldTextInput).type(domainEditedName)
     cy.get(selectors.domains.cnamesField).clear()
-    cy.get(selectors.domains.cnamesField).type(`${domainName}-edit.edge.app`)
+    cy.get(selectors.domains.cnamesField).type(`wwww.${domainName}.com`)
```
[the remark this answers is not in the document]
Comment (pauloSF0) [reply cypress/e2e/domains/edit-domain.cy.js:64]: Boa pergunta
Eu alterei o valor de cnames porque a API não estava aceitando o cname enviado.

[the remark this answers is not in the document]
Comment (pauloSF0) [reply cypress/e2e/domains/edit-domain.cy.js:64]: Vou tentar pegar o erro que eu estava recebendo

[the remark this answers is not in the document]
Comment (pauloSF0) [reply cypress/e2e/domains/edit-domain.cy.js:64]: cannot_use_cnames: domain251124165758171.edge.app

**Review by PatrickMenoti — APPROVED** [review]

**Review by lucasmendes21 — APPROVED** [review]

**Comment by vinigfer** [comment]
We need to wait BF Freezing to be over to merge this. Otherwise test execution will start to break for others

**Pull request merged**